traveller.yaml

```diff
@@ -1221,43 +1158,30 @@ definitions:
       tokenProof:
         $ref: '#/definitions/tokenProof'
 
-  purchaseRequest:
+  transactionUpdate:
     type: object
     properties:
-      pid:
-        $ref: '#/definitions/pid'
-      productId:
-        description: Product ID to purchase
-        type: string
+      transactionCommitted:
+        description: Set to true to finalise transaction.
+        type: boolean
+      transactionCancelled:
+        description: Set to true to cancel transaction.
+        type: boolean
       transactionReference:
         description: Requestors transaction reference
         type: string
 
```

Comment:
How should transactionUpdate.transactionReference be used?

Reply:
When you create a wallet transaction, you can specify a `transactionReference` in the `walletTransactionRequest`. Having it patchable in `transactionInformationUpdate` was to allow it to be set later (when committing) if needed.